Applications/MmrMobile/Client/MmrMobileClient.cs

```diff
@@ -162,6 +162,37 @@ public async Task<MmrNodeData> GetTimeSeriesFromMachine(string machineNumber, st
         }
         #endregion
 
+        #region machinedata
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<AlertEvent>> GetEventSeriesFromMachine(string machineNumber, DateTime from, DateTime to, int take = 1000, int skip = 0)
+        {
+            string url = $"/api/mmr-mobile/eventdata/machines/{machineNumber}/history?from={Uri.EscapeDataString(from.ToString("o", CultureInfo.InvariantCulture))}&to={Uri.EscapeDataString(to.ToString("o", CultureInfo.InvariantCulture))}&take={take}&skip={skip}";
+
```

Comment:
Could be done much shorter equal to the material statistics.

 var url = $"/api/intelliDivide/statistics/material?from={from:s}&to={to:s}&take={take}&skip={skip}";

 return await RequestEnumerable<MaterialEfficiency>(new Uri(url, UriKind.Relative));

Reply:
At the moment, the helios-data (itemdata, not events) have these endpoints:
/api/mmr-mobile/machinedata/machines/{machineNumber}/nodes
/api/mmr-mobile/machinedata/machines/{machineNumber}/nodes/{node}
/api/mmr-mobile/machinedata/machines/{machineNumber}/nodes/{node}?timestamp=...

So I only changed "machinedata" to "eventdata" or perhaps better "alerteventdata"

Shall we then change all endpoints? And move the machinenumber into the body?